site/releases.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Apache BookKeeper Releases
+layout: community
+---
+
+## Download
+
+{{ site.stable_release }} is latest release. The current stable version is {{ site.stable_release }}.
+
+Releases are available to download from Apache mirrors: [Download](http://www.apache.org/dyn/closer.cgi/bookkeeper)
+
+You can verify your download by following these [procedures](http://www.apache.org/info/verification.html) and using these [KEYS](https://dist.apache.org/repos/dist/release/bookkeeper/KEYS).
+
+If you want to download older releases, they are available in the [Apache archive](http://archive.apache.org/dist/bookkeeper/).
+
+## Getting Started
+
+Once you've downloaded a BookKeeper release, instructions on getting up and running with a standalone cluster that you can run your laptop can be found
+in [Run BookKeeper locally]({{ site.baseurl }}/docs/getting-started/run-locally).
+
+If you need to connect to an existing BookKeeper cluster using an officially supported client, see client docs for these languages:
+
+Client Guide | API docs
+:------------|:--------
+[The BookKeeper Client]({{ site.baseurl }}/docs/api/ledger-api) | [Javadoc]({{ site.baseurl }}/javadoc)
+[The DistributedLog Library]({{ site.baseurl }}/docs/api/distributedlog-api) | [Javadoc](https://distributedlog.io/docs/latest/api/java)
+
+## News
+
+### 16 May, 2016: release 4.4.0 available
```

Comment:
I think it'd be better to generate this section using a template, but I can do that later